### PR TITLE
Sprint4 part3

### PR DESCRIPTION
**Pull request by kamilkoterba**

### CODE REVIEW

**Comment by kvas-damian** [comment]
content of `.icon-chevron-gray` is included in output css 7 times.
We should include it only once.
The same rule is valid for other base64 files

File: wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/**
```
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js:1]: I think we are used to camelCase for file names:
https://github.com/Wikia/guidelines/blob/master/JavaScript/CodingConventions.md#js-files

File: wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+	// Initiates click events
+	var initClickEvent = function() {
+		// Click on active tab
+		$('.nav-tabs .active').click(doTheScroll);
+		// Click on skip banner button
+		$('#site-header .scroll-down').click(skipBannerScroll);
+	};
+
+	var doTheScroll = function(e) {
```
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js:16]: e => event
e is not too descriptive

File: wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		e.preventDefault();
+		var elementClicked =$(this);
```
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js:18]: one var per function

File: wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js (hunk 1 above)
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/js/plugins/scroll_to_content.js:1]: please refer to our coding conventions
unless this is a 3rd party - in this case please mark somehow files like that so we won't review them

File: wp-content/themes/wikia-careers/assets/less/wikia-careers/buttons.less [hunk 4]
```diff
@@ -57,3 +47,25 @@
...
+	&:focus,
+	&:active,
+	&:visited {
+		border: 0;
+		outline: none;
+	}
+	.scroll-down::-moz-focus-inner {
+		border: 0;
+	}
+}
```
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/less/wikia-careers/buttons.less:71]: \n

File: wp-content/themes/wikia-careers/assets/less/wikia-careers/buttons.less [hunk 5]
```diff
@@ -57,3 +47,25 @@
...
 	}
 }
+
+.scroll-down {
+	width: 100px;
+	height: 50px;
+	background-position: 50% 50%;
+	background-color: transparent;
+	border: 0;
+	transition: all .2s linear;
```
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/less/wikia-careers/buttons.less:57]: what exactly do you animate here?

File: wp-content/themes/wikia-careers/assets/less/wikia-careers/page-job-offers.less [hunk 6]
```diff
@@ -79,12 +72,20 @@ h1+.offer-item{
...
 		margin-bottom: 0;
 		margin-top: 0;
-		transition: all .5s linear;
 		margin-bottom: 0px;
 		margin-top: 0px;
 	}
 }
+
+.job-offers-list {
+	.padding-section-vertical();
```
Comment (hakubo) [inline wp-content/themes/wikia-careers/assets/less/wikia-careers/page-job-offers.less:86]: \n

**Pull request merged**